gov-notify-stub/src/MockGovNotifyHandler.ts

```diff
@@ -41,6 +41,18 @@ class MockGovNotifyHandler implements LambdaInterface {
 				logger.info("PARSED EMAIL", payloadParsed.email_address);
 				logger.info("Starting GovNotifyRequestProcessor");
 				return await GovNotifyRequestProcessor.getInstance(logger, metrics).mockSendEmail(payloadParsed.email_address);
```

Comment:
As mentioned could use different processor instead of updating the existing one and manages load as well. Rename the old processor to GovNotifySendEmailRequestProcessor and likewise for the new one.